test/buildah-bud/buildah-tests.diff

```diff
@@ -21,10 +21,17 @@ index aab3f72e..6e3b0eb5 100644
 +    PODMAN_NATIVE="${PODMAN_BINARY} ${ROOTDIR_OPTS} ${PODMAN_REGISTRY_OPTS}"
 +    if [[ -n "$REMOTE" ]]; then
 +        PODMAN_NATIVE="${PODMAN_BINARY%%-remote} ${ROOTDIR_OPTS} ${PODMAN_REGISTRY_OPTS}"
++        if [[ -n "$PODMAN_SERVER_LOG" ]]; then
++            (
++                echo "---------------------------------"
++                echo "- bats test ${BATS_TEST_NUMBER} : ${BATS_TEST_NAME}"
++            ) >> $PODMAN_SERVER_LOG
++        fi
++
 +        # static CONTAINERS_CONF needed for capabilities test. As of 2021-07-01
 +        # no tests in bud.bats override this; if at some point any test does
 +        # so, it will probably need to be skip_if_remote()d.
-+        env CONTAINERS_CONF=${CONTAINERS_CONF:-$(dirname ${BASH_SOURCE})/containers.conf} $PODMAN_NATIVE system service --timeout=0  &
++        env CONTAINERS_CONF=${CONTAINERS_CONF:-$(dirname ${BASH_SOURCE})/containers.conf} $PODMAN_NATIVE system service --timeout=0 &>>${PODMAN_SERVER_LOG:-/dev/stderr} &
```

Comment:
```suggestion
+        env CONTAINERS_CONF=${CONTAINERS_CONF:-$(dirname ${BASH_SOURCE})/containers.conf} $PODMAN_NATIVE system service --timeout=0 &>>${$PODMAN_SERVER_LOG:-/dev/stderr} &
```

@edsantiago could this fix the issue?  Not sure though ... bash isn't my strong suit.

Reply:
That's actually a syntax error.  The `${ VAR colon-minus default}` means to use stderr if `PODMAN_SERVER_LOG` is unset:
```console
$ echo ${PODMAN_SERVER_LOG:-/dev/stderr}
/dev/stderr
$ PODMAN_SERVER_LOG=/tmp/foo
$ echo ${PODMAN_SERVER_LOG:-/dev/stderr}
/tmp/foo
```